src/archivist/fetcher/exports.js

```diff
@@ -0,0 +1,5 @@
+import fetcher from './index.js';
```

Comment:
```suggestion
export { default as fetcher } from './index.js'
```

And remove `export default fetcher;`